### PR TITLE
f-takeawaypay-activation@2.4.0 - Add component and accessibility tests

### PR DESCRIPTION
**Pull request by CarlosOrtegaGonzalez2019**
v2.4.0
------------------------------
*October 11, 2021*

### Changed
- Component tests
- Accessibility tests
- Visual regression tests

## Browsers Tested
- [x] Chrome (latest)

### CODE REVIEW

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js [hunk 1]
```diff
@@ -1,13 +1,169 @@
...
+        expect(pathname).toEqual(registerPath);
+    });
+});
+
+describe('f-takeawaypayActivation component tests - logged in as a guest', () => {
+    beforeEach(() => {
+        // Arrange
+        takeawayPayComponent = new TakeawaypayActivation();
+        takeawayPayComponent
+        .withQuery('knob-Authentication', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbSIsImNyZWF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkzMDAwMFoiLCJzdWIiOiIxODcwMzA5MyIsIm5hbWUiOiJKb2UgQmxvZ2dzIiwiZ2xvYmFsX3VzZXJfaWQiOiJVN05SQWxXQWc1ek9kc2RSZ2Y3bmtUeW9pOTBYRW89IiwiZ2l2ZW5fbmFtZSI6IkpvZSIsImZhbWlseV9uYW1lIjoiQmxvZ2dzIiwicm9sZSI6Ikd1ZXN0IiwiaWF0IjoxNjE1NDY5NTE2fQ.d-dLpZM8vi7IR1GdqZI9IPzUcjnidU-7qO62B3Nfk6I')
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Move this into a constants file that's imported into the spec.

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js [hunk 2]
```diff
@@ -1,13 +1,169 @@
...
+    });
+});
+
+describe('f-takeawaypayActivation component tests - authenticated', () => {
     beforeEach(() => {
-        takeawaypayActivation.load();
+        // Arrange
+        takeawayPayComponent = new TakeawaypayActivation();
+        takeawayPayComponent
+            .withQuery('knob-Authentication', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbSIsImNyZWF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkzMDAwMFoiLCJuYW1lIjoiSm9lIEJsb2dncyIsImdsb2JhbF91c2VyX2lkIjoiVTdOUkFsV0FnNXpPZHNkUmdmN25rVHlvaTkwWEVvPSIsImdpdmVuX25hbWUiOiJKb2UiLCJmYW1pbHlfbmFtZSI6IkJsb2dncyIsImlhdCI6MTYxNTQ2OTUxNn0.VapH6uHnn4lHIkvN_mS9A9IVVWL0YPNE39gDDD-l7SU')
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: ditto

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js [hunk 3]
```diff
@@ -1,13 +1,169 @@
...
+        expect(pathname).toEqual(homePath);
     });
 });
+
+describe('f-takeawaypayActivation component tests - error page', () => {
+    beforeEach(() => {
+        // Arrange
+        takeawayPayComponent = new TakeawaypayActivation();
+        takeawayPayComponent
+            .withQuery('knob-Authentication', 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbSIsImNyZWF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkzMDAwMFoiLCJuYW1lIjoiSm9lIEJsb2dncyIsImdsb2JhbF91c2VyX2lkIjoiVTdOUkFsV0FnNXpPZHNkUmdmN25rVHlvaTkwWEVvPSIsImdpdmVuX25hbWUiOiJKb2UiLCJmYW1pbHlfbmFtZSI6IkJsb2dncyIsImlhdCI6MTYxNTQ2OTUxNn0.VapH6uHnn4lHIkvN_mS9A9IVVWL0YPNE39gDDD-l7SU')
```
Comment (siggerzz) [inline packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: ditto

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js (hunk 1 above)
Comment (CarlosOrtegaGonzalez2019) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Done

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js (hunk 2 above)
Comment (CarlosOrtegaGonzalez2019) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Done

File: packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js (hunk 3 above)
Comment (CarlosOrtegaGonzalez2019) [reply packages/components/organisms/f-takeawaypay-activation/test/component/f-takeawaypayActivation.component.desktop.spec.js]: Done

**Review by siggerzz — APPROVED** [review]

**Review by oleksiishevchukje — APPROVED** [review]

**Pull request merged**